utils/verify-package.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euo pipefail
+
+# everything merged to 'main' is published to this URL
+packageUrl="https://api-frameworks.zeit.sh/tarballs/vercel.tgz"
```

Comment:
We'd just have to be careful to make sure latest main has the Vercel deployment completed already. This will probably usually be the case already, but something to keep in mind.

An alternative would be to wait for the Vercel deployment of the associated commit to complete and then use the unique URL from that, but that would slow down the Publish step quite a bit.